templates/default/aurora-security.erb

```diff
@@ -0,0 +1,7 @@
+[users]
+<% @admins.sort.each do |key, value| %>
+<%= key %> = <%= value %>, admin
+<% end %>
+
+[roles]
+admin = *
```

Comment:
OPT: roles might be configurable (according to http://aurora.apache.org/documentation/latest/operations/security/#using-an-ini-file-to-define-security-controls, we can set different types of roles to the groups, etc.)
Adding a `roles` hash can be sufficient to let the end-user customize that part.